<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainForm</class>
 <widget class="QWidget" name="mainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>365</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>688</width>
    <height>365</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>688</width>
    <height>365</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>[COMSYS Tool] S1 Report Generator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QDateEdit" name="date_start">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QCalendarWidget QToolButton {
  	height: 20px;
  	width: 60px;
  	color: white;
  	font-size: 12px;
  	icon-size: 20px, 20px;
  	background-color: black;
  }
  QCalendarWidget QMenu {
  	width: 60px;
    left: 20px;
  	color: white;
  	font-size: 12px;
  	background-color: rgb(100, 100, 100);
  }
  QCalendarWidget QSpinBox { 
  	width: 60px; 
  	font-size:12px; 
  	color: white; 
  	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #cccccc, stop: 1 #333333); 
  	selection-background-color: rgb(136, 136, 136);
  	selection-color: rgb(255, 255, 255);
  }
  QCalendarWidget QSpinBox::up-button { subcontrol-origin: border;  subcontrol-position: top right;  width:20px; }
  QCalendarWidget QSpinBox::down-button {subcontrol-origin: border; subcontrol-position: bottom right;  width:20px;}
  QCalendarWidget QSpinBox::up-arrow { width:20px;  height:20px; }
  QCalendarWidget QSpinBox::down-arrow { width:20px;  height:20px; }
   
  /* header row */
  QCalendarWidget QWidget { alternate-background-color: rgb(128, 128, 128); }
   
  /* normal days */
  QCalendarWidget QAbstractItemView:enabled 
  {
  	font-size:12px;  
  	color: rgb(180, 180, 180);  
  	background-color: black;  
  	selection-background-color: rgb(64, 64, 64); 
  	selection-color: rgb(0, 255, 0); 
  }
   
  /* days in other months */
  /* navigation bar */
QCalendarWidget QWidget#qt_calendar_navigationbar
{ 
  background-color: black; 
}

QCalendarWidget QAbstractItemView:disabled 
{ 
color: rgb(64, 64, 64); 
}</string>
      </property>
      <property name="maximumDate">
       <date>
        <year>2111</year>
        <month>1</month>
        <day>6</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2010</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="maximumSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="lineWidth">
       <number>7</number>
      </property>
      <property name="text">
       <string>~</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateEdit" name="date_end">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">
QCalendarWidget QToolButton {
  	height: 20px;
  	width: 60px;
  	color: white;
  	font-size: 12px;
  	icon-size: 20px, 20px;
  	background-color: black;
  }
  QCalendarWidget QMenu {
  	width: 60px;
    left: 20px;
  	color: white;
  	font-size: 12px;
  	background-color: rgb(100, 100, 100);
  }
  QCalendarWidget QSpinBox { 
  	width: 60px; 
  	font-size:12px; 
  	color: white; 
  	background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop: 0 #cccccc, stop: 1 #333333); 
  	selection-background-color: rgb(136, 136, 136);
  	selection-color: rgb(255, 255, 255);
  }
  QCalendarWidget QSpinBox::up-button { subcontrol-origin: border;  subcontrol-position: top right;  width:20px; }
  QCalendarWidget QSpinBox::down-button {subcontrol-origin: border; subcontrol-position: bottom right;  width:20px;}
  QCalendarWidget QSpinBox::up-arrow { width:20px;  height:20px; }
  QCalendarWidget QSpinBox::down-arrow { width:20px;  height:20px; }
   
  /* header row */
  QCalendarWidget QWidget { alternate-background-color: rgb(128, 128, 128); }
   
  /* normal days */
  QCalendarWidget QAbstractItemView:enabled 
  {
  	font-size:12px;  
  	color: rgb(180, 180, 180);  
  	background-color: black;  
  	selection-background-color: rgb(64, 64, 64); 
  	selection-color: rgb(0, 255, 0); 
  }
   
  /* days in other months */
  /* navigation bar */
QCalendarWidget QWidget#qt_calendar_navigationbar
{ 
  background-color: black; 
}

QCalendarWidget QAbstractItemView:disabled 
{ 
color: rgb(64, 64, 64); 
}</string>
      </property>
      <property name="maximumDate">
       <date>
        <year>2110</year>
        <month>12</month>
        <day>31</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2010</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
      <property name="date">
       <date>
        <year>2010</year>
        <month>1</month>
        <day>2</day>
       </date>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>579</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QComboBox" name="cmb_account">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>175</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>175</width>
        <height>30</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>169</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	background-color: white;
	border: 1px solid rgb(0,0,0);
	border-radius: 20px;
}
QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 rgb(255, 255, 255), stop: 0.4 rgb(255, 255, 255),
                                 stop: 0.5 rgb(255, 255, 255), stop: 1.0 rgb(255, 255, 255));
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 rgb(255, 255, 255), stop: 0.4 rgb(255, 255, 255),
                                stop: 0.5 rgb(255, 255, 255), stop: 1.0 rgb(255, 255, 255));
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="lineWidth">
       <number>7</number>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmb_site">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>175</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>175</width>
        <height>30</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>169</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	background-color: white;
	border: 1px solid rgb(0,0,0);
	border-radius: 20px;
}
QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 rgb(255, 255, 255), stop: 0.4 rgb(255, 255, 255),
                                 stop: 0.5 rgb(255, 255, 255), stop: 1.0 rgb(255, 255, 255));
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 rgb(255, 255, 255), stop: 0.4 rgb(255, 255, 255),
                                stop: 0.5 rgb(255, 255, 255), stop: 1.0 rgb(255, 255, 255));
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="lineWidth">
       <number>7</number>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmb_group">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>175</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>175</width>
        <height>30</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>169</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>맑은 고딕</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	background-color: white;
	border: 1px solid rgb(0,0,0);
	border-radius: 20px;
}
QComboBox:editable {
    background: white;
}

QComboBox:!editable, QComboBox::drop-down:editable {
     background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                 stop: 0 rgb(255, 255, 255), stop: 0.4 rgb(255, 255, 255),
                                 stop: 0.5 rgb(255, 255, 255), stop: 1.0 rgb(255, 255, 255));
}

/* QComboBox gets the &quot;on&quot; state when the popup is open */
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 rgb(255, 255, 255), stop: 0.4 rgb(255, 255, 255),
                                stop: 0.5 rgb(255, 255, 255), stop: 1.0 rgb(255, 255, 255));
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>조회 날짜 지정</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>조회 스코프 지정</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="text">
    <string>Report Generator Tool</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_expConfirm">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>250</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color: white;
background-color: black;
border: 2px solid rgb(0, 0, 0);
border-radius: 5px;
}
QPushButton::pressed{
color: black;
background-color: white;
border: 2px solid rgb(0, 0, 0);
border-radius: 5px;
}
QPushButton::disabled{
color: rgb(229, 229, 229);
background-color: rgb(204, 204, 204);
border: 2px solid rgb(204, 204, 204);
border-radius: 5px;
}
</string>
   </property>
   <property name="text">
    <string>보고서 생성</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_export">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>631</width>
     <height>60</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>650</width>
     <height>60</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_version">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_filepath">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>210</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>70</width>
     <height>30</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>8</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
color: white;
background-color: black;
border: 2px solid rgb(0, 0, 0);
border-radius: 5px;
}
QPushButton::pressed{
color: black;
background-color: white;
border: 2px solid rgb(0, 0, 0);
border-radius: 5px;
}
QPushButton::disabled{
color: rgb(229, 229, 229);
background-color: rgb(204, 204, 204);
border: 2px solid rgb(204, 204, 204);
border-radius: 5px;
}
</string>
   </property>
   <property name="text">
    <string>경로 변경</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>리포트 저장 경로</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_filepath">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>550</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>550</width>
     <height>30</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>맑은 고딕</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
	background-color: white;
	border: 1px solid rgb(0,0,0);
	border-radius: 20px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
   <property name="clearButtonEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>* 선택한 경로에 s1_ReportGenerator 폴더가 생성됩니다.</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="prBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>650</width>
     <height>10</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>688</width>
     <height>10</height>
    </size>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
   <property name="format">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
